### ISSUE TITLE
OWID - World map on share of people vaccinated against COVID-19,

### ISSUE THREAD

**Issue by jravenel** [jupyter-naas/awesome-notebooks#210]
Objective: create a world map inspired by the graphs and data available on: https://ourworldindata.org/covid-vaccinations

Dataset: https://github.com/owid/covid-19-data/tree/master/public/data/vaccinations/country_data

**Comment by WalquerX** [comment]
hey @jravenel I would like to work on this with you

**Comment by jravenel** [comment]
> hey @jravenel I would like to work on this with you

Hey @WalquerX I just added you to the contribution team. You can now use your Naas account, clone awesome-notebooks, create new branch, create OWID folder, and start working on it. 

Feel free to ask if you need any guidance.

**Comment by WalquerX** [comment]
Hi @jravenel, I would appreciate some guidance. About it do I have to do it on python or can i use another language like R? Also what do you mean by OWID folder? for the dataset? should I work as if the data is in local and not write some code to download it directly form the source web page?

**Comment by jravenel** [comment]
> Hi @jravenel, I would appreciate some guidance. About it do I have to do it on python or can i use another language like R? Also what do you mean by OWID folder? for the dataset? should I work as if the data is in local and not write some code to download it directly form the source web page?

Yes, we do everything in Python. Is that ok for you?
In naas awesome-notebooks you will need to create a folder OWID as it's the source of the data, then write the notebook. 

Please check this video to learn how to contribute step by step : https://www.youtube.com/watch?v=jmC5VDN-MOI
If needed, feel free to book a slot in my calendar https://calendly.com/jeremyravenel 

Hope it helps :)
Thanks again for your interest in Naas.

**Comment by WalquerX** [comment]
Yes, python is fine.  incase I have any difficulty I will reach out.

**Comment by OketunjiOludolapo** [comment]
Hi. Can i help you guys on this and do a PR?

**Comment by WalquerX** [comment]
> Hi. Can i help you guys on this and do a PR?

Hi, I could not make progress on it. Go with it.
